### PR TITLE
Fix migrated content images

### PR DESCRIPTION
**Pull request by PerVillalva**
@mnmkng I had forgotten that we use a specific path structure for images in the docs. So, I've now fixed the image paths in the recently migrated articles from the Academy and also changed them to 'webp' images instead of 'png.'

### CODE REVIEW

**Comment by B4nan** [comment]
Please rebase so we can merge, I'd like to deploy the docs this afternoon. If you expect reviews from Ondra, try pinging him on slack, I believe he is no longer subscribed to GH notifications.

File: content/academy/node_js/apify_free_google_serp_api.md [hunk 1]
```diff
@@ -8,8 +8,12 @@ paths:
 
 So you need to regularly grab SERP data about your target keywords? Apify provides a free SERP API that includes organic search, ads, people also ask, etc. Free Apify accounts come with 30 days of proxy trial and $5 of credit. To get started, head over to the [Google Search Results Scraper](https://apify.com/apify/google-search-scraper) page and click the `Try me` button. You'll be taken to a page where you can enter the search query, region, language and other settings.
 
+<<<<<<< HEAD
```
Comment (B4nan) [inline content/academy/node_js/apify_free_google_serp_api.md:11]: these are marking the conflict you need to resolve, you dont wanna commit them.

Comment (PerVillalva) [reply content/academy/node_js/apify_free_google_serp_api.md:11]: Yeah, I'm fixing them. I will push the fix in a soon.

**Comment by PerVillalva** [comment]
@B4nan, I just resolved the conflicts. I will ping Ondra now.

**Comment by B4nan** [comment]
BTW the new docs will use source images (and optimize automatically), and regular links, so this won't be a problem anymore.

**Review by mnmkng — APPROVED** [review]

**Pull request merged**